docs/user-guide/pods.md

```diff
@@ -50,42 +50,92 @@ Documentation for other releases can be found at
 
 <!-- END MUNGE: GENERATED_TOC -->
 
-In Kubernetes, rather than individual application containers, _pods_ are the smallest deployable units that can be created, scheduled, and managed.
+_pods_ are the smallest deployable units that can be created and managed in
```

Comment:
_pods_ cause a program to run.  If you don't know anything about kbernetes, you might not get "compute" out of this sentence (or the old one).